src/platform_impl/linux/wayland/event_loop.rs

```diff
@@ -323,24 +323,80 @@ impl SeatManager {
 
     fn remove_seat(&mut self, id: u32) {
         use self::wl_seat::RequestsTrait as SeatRequests;
+<<<<<<< HEAD:src/platform_impl/linux/wayland/event_loop.rs
```

Comment:
Is this intentional?

Reply:
This was an othersight that was fixed in #790

Reply:
Yeah. I just wonder how CI didn't find it.

Reply:
CI didn't pass at all for the other platforms than windows, which were not ported yet in this PR.